backend/comment-section.go

```diff
@@ -23,11 +23,9 @@ import (
 )
 
 const (
-	COMMENT_SAMPLE_PATH         = "/samples_templates/comment_section/"
-	COMMENT_SAMPLE_PATH_PREVIEW = COMMENT_SAMPLE_PATH + "preview/"
-	COMMENT_COOKIE_NAME         = "ABE_LOGGED_IN"
-	SUBMIT_COMMENT_XHR          = "submit-comment-xhr"
-	USER                        = "Charlie"
+	COMMENT_SAMPLE_PATH = "/" + CATEGORY_SAMPLE_TEMPLATES + "/comment_section/"
```

Comment:
maybe CATEGORY_SAMPLE_TEMPLATES -> CATEGORY_SAMPLES_TEMPLATES given the original value contains the s?